.github/workflows/build-and-push-nosql-sec-image.yml

```diff
@@ -0,0 +1,52 @@
+name: Build and publish NoSQL container image to GitHub Container Registry
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'NoSQL/ce-sec/*'
+      - '.github/workflows/build-and-push-nosql-image.yml'
+  workflow_dispatch:
+
+env:
+  IMAGE_NAME: nosql
+  TAG: ce
+
+jobs:
+  push:
+    name: Build and push NoSQL ce image
+
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push'
```

Comment:
This will limit the workflow to only trigger on `push` even though you have `paths` configured in the `on` section. If you only want to trigger a build on push to main, just remove the `paths` section from the `on` configuration above.

Reply:
I didn't understand, I am reusing the previous script